### ISSUE TITLE
Virtualized table doesn't render all elements

### ISSUE THREAD

**Issue by gioeau** [iTwin/iTwinUI-react#718]
#### Describe the bug (actual behavior)
When table is created but not shown, only 20 rows are rendered afterwards. Scrolling the table doesn't make any more rows appear. Only occurs when virtualization is enabled.

#### Expected behavior
Table should render all rows in viewport.

#### Reproduction
https://codesandbox.io/s/itwinui-react-minimal-example-forked-zxyp3w?file=/src/App.tsx

##### Steps to reproduce
1. Press 'Show' button.
2. Scroll
3. See that no more rows are rendered

#### Additional information
Looks like in VirtualScroll.js `childHeight` is initially set to 0 and doesn't update. It depends on `visibleChildren`, which is calculated using `visibleNodeCount`. The problem is, if `childHeight == 0`, `visibleNodeCount` doesn't update either. Would call this a circular dependency?

**Comment by veekeys** [comment]
Yeah, Virtual Scroll needs elements to be in the DOM for initial calculations, but if you hide those, it might fail.
We could have a look at it if there is a nice way to ensure it is recalculated.. Just it is risky, not to overdo it.

**Comment by rohan-kadkol** [comment]
Hello @gioeau, I was trying to work on this task but was unfortunately unable to reproduce this behavior. For instance, after visiting your codesandbox link, I could not find the `Show` button you were referring to. Maybe I am doing something wrong in reproducing this problem, but could you please clarify what was the `Show` button you were referring to? If applicable, could you also include any screenshots that might help? Thanks.

**Comment by GintV** [comment]
@rohan-kadkol Codesandbox example was fixed

**Comment by gioeau** [comment]
Hi @rohan-kadkol I think @GintV meant to say that codesandbox example now reproduces the issue. You can see in the sandbox there are 1000 elements, but when the "Show" button is pressed only a few are loaded.

**Comment by rohan-kadkol** [comment]
> Hi @rohan-kadkol I think @GintV meant to say that codesandbox example now reproduces the issue. You can see in the sandbox there are 1000 elements, but when the "Show" button is pressed only a few are loaded.

Ohh, sorry, I think I misunderstood. Perfect, checking it now.

**Comment by rohan-kadkol** [comment]
Fixed in [v1.46.0](https://github.com/iTwin/iTwinUI-react/releases/tag/v1.46.0)! 🚀
